doc/container.md

```diff
@@ -266,6 +266,26 @@ with several versions of the Linux kernel, including at least kernel versions up
 If a kernel upgrade does not help, please use a different access mode for NFS-mounted directories,
 such as `--hidden-dir` or `--read-only-dir`.
 
+#### Problems due to default user namespaces restrictions on Ubuntu 24.04 (and possibly later versions)
```

Comment:
In this section, we try to show the error message in the heading, such that it is easy to find. Could you please also do this?